### PR TITLE
team-list: establish categorization team

### PR DESCRIPTION
**Pull request by tomodachi94**
> [!IMPORTANT]  
> If you want to be on the team initially, please comment by 2024-12-15 (we will remove non-responsive initial team members after this deadline). We can add interested people to the team later, if they are still interested.

Per RFC 0146:
https://github.com/NixOS/rfcs/blob/master/rfcs/0146-meta-categories.md#categorization-team

> "a team entrusted with authority to manage issues related to categorization and carry their corresponding duties"

I added people to the team who were pinged in the below tracking issue, sorted alphabetically; the list of members is not set in stone.

Tracking: https://github.com/NixOS/nixpkgs/issues/360727

I requested reviews from the shepherd team for the RFC, as well as the creator of the above tracking issue.

I think it's reasonable to make **shepherd reviews (at least from Raskin, the lead) blocking.** Other reviews are, of course, appreciated.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: maintainers/team-list.nix
Comment (tomodachi94) [inline maintainers/team-list.nix:144]: I could not find an entry in `maintainer-list.nix` for this person.

Comment (tomodachi94) [reply maintainers/team-list.nix]: I removed @pyrotelekinetic per above. Could you add yourself to `maintainers/maintainer-list.nix`?

**Comment by SigmaSquadron** [comment]
I'd love to help with the manual labeling of packages! Let me know when we can start tagging.

**Comment by tomodachi94** [comment]
Added :)

**Comment by tomodachi94** [comment]
Created a [GitHub team](https://github.com/orgs/NixOS/teams/categorization) and linked it to the team created by this PR.

File: maintainers/team-list.nix
Comment (pyrotelekinetic) [reply maintainers/team-list.nix]: PR'd: https://github.com/NixOS/nixpkgs/pull/362952

Comment (tomodachi94) [reply maintainers/team-list.nix:144]: Rebased and readded to the team list. Thank you ❤️

**Review by getchoo — APPROVED** [review]
Excited to see what comes out of this :)

**Review by Aleksanaa — APPROVED** [review]

**Review by SigmaSquadron — APPROVED** [review]

**Comment by 7c6f434c** [comment]
Seems fine to me — but is everyone in the proposed team review-requested (or has already reacted here)? It looks like not enough review requests.

**Comment by tomodachi94** [comment]
> Seems fine to me — but is everyone in the proposed team review-requested (or has already reacted here)? It looks like not enough review requests.

I requested most of the reviews but missed one. If I recall correctly, the people added here have already expressed interest in joining the team, but let's wait for approvals from everyone before we merge this.

**Review by tr3foil — APPROVED** [review]

**Review by SigmaSquadron — APPROVED** [review]

**Review by natsukium — APPROVED** [review]
Thank you!

**Comment by tomodachi94** [comment]
Would it be reasonable to remove anyone who doesn't respond by the start of next week (2024-12-15)? I think we're supposed to have the approvals of everyone listed on this team, but I'm not confident we'll get a response from everyone.

**Review by Aleksanaa — APPROVED** [review]

**Review by Lyndeno — APPROVED** [review]

**Comment by Lyndeno** [comment]
I look forward to seeing what we can accomplish here

**Comment by tomodachi94** [comment]
Currently just waiting on a response from @fgaz.

**Comment by philiptaron** [comment]
I'll be on the team, if you'll have me. Information architecture is quite important.

**Review by philiptaron — APPROVED** [review]

**Review by fgaz — APPROVED** [review]

**Comment by Aleksanaa** [comment]
Merging this as we have gathered all approvals 🚀

**Pull request merged**
